get-lm-desk.sh

```diff
@@ -634,39 +569,21 @@ then
     pull_models
 fi
 
-#########################################################
-# Install continue into VS Code if VS Code is installed #
-#########################################################
-have_continue=0
-if [ "$code_bin" != "" ] && yes_no_prompt "Install continue?"
-then
-    install_continue
-    have_continue=1
-fi
-
-
-##########################################
-# Install jq if needed for configuration #
-##########################################
-continue_config="$HOME/.continue/config.json"
-if [ "$jq_bin" == "" ] && [ "$have_continue" == "1" ] && [ -f $continue_config ] && yes_no_prompt "Install jq?"
+############################
+# Install uv if needed #
+############################
+if [ "$uv_bin" == "" ] && yes_no_prompt "Install uv?"
 then
-    install_jq
+    install_uv
     report_installed
 fi
 
-################################################
-# Configure continue to use the desired models #
-################################################
-if [ "$have_continue" ] && [ -f $continue_config ] && [ "$jq_bin" != "" ] && yes_no_prompt "Configure continue?"
+
+############################
+# Run everything 
```

Comment:
I think we want to separate the running part from the installing part

Reply:
I think the reason why Veronique does the installation and the run because this was meant to be for the layman and we want to have them run one script and be able to start using it. If that is not the goal, how do you see it working? Would they run one script to install and then be provided some output that gives them commands on how to run openwebui?